docs/docs/cli/creating-data-stores.md

```diff
@@ -0,0 +1,64 @@
+# Defining Data Stores as Text Files
+You might have multiple tracetest instances that need to be connected to the same data stores. An easy way of sharing the configuration is by using a configuration file that can be applied to your tracetest instance.
+
+### Jaeger
+```yaml
+type: DataStore
+spec:
+    name: development
+    type: jaeger
+    isDefault: true
+    jaeger:
+        endpoint: 127.0.0.1:16685
+        tls:
+            insecure: true
+```
+
+### Tempo
+```yaml
+type: DataStore
+spec:
+  name: Grafana Tempo
+  type: tempo
+  isDefault: true
+  tempo:
+    endpoint: tempo:9095
+    tls:
+      insecure: trues
+```
+
+### OpenSearch
+```yaml
+type: DataStore
+spec:
+  name: OpenSearch Data Store
+  type: openSearch
+  isDefault: true
+  opensearch:
+    addresses:
+      - http://opensearch:9200
+    index: traces
+```
+
+### SignalFX
+```yaml
+type: DataStore
+spec:
+  name: SignalFX
+  type: signalFx
+  isDefault: true
+  signalFx:
+    realm: us1
+    token: mytoken
+```
+
+## Apply configuration
+
+To apply the configuration, you need a [configured CLI](./configuring-your-cli.md) pointed to the instance you want to apply the data store. Then you just have to enter:
+
+```
+tracetest data-store apply -f my/data-store/file/location.yaml
+```
+
+## Additional information
```

Comment:
```suggestion
## Additional Information
```

Reply:
Do we have a way to export the data store configuration yet? If so, we should document it as the best way to build a configuration file to be used elsewhere to import a configuration.

Reply:
No, we don't have it yet.